platform/user-grant/grant.go

```diff
@@ -166,11 +173,71 @@ func (c *Config) CertGen(key crypto.PublicKey, user string) ([]byte, error) {
 	return certutil.FinishCertificate(certTemplate, c.IssuerCert, key, c.IssuerKey)
 }
 
+func (c *Config) CreateNamespace(namespace string) error {
```

Comment:
Can we add at least one check that just makes sure that `namespace` starts with `user-`?  I know that as written, this code only ever does just that, but this is a pretty critical invariant.